pyproject.toml

```diff
@@ -69,6 +69,7 @@ dependencies = [
     "tenacity~=8.2.2",
     "terminaltables~=3.1.10",
     "traceback-with-variables~=2.0.4",
+    "typing_extensions", # included by Pydantic, adding here since it's now being directly used in code
```

Comment:
let's at least pin the major version. They do have breaking changes.

Reply:
How about adding `'typing-extensions>=4.6.1'` as defined in Pydantic's toml?

Reply:
I'd still prefer an additional `<5.0.0`. These things randomly break installations everywhere and the only fix is an immediate patch release. :smiling_face_with_tear: